### ISSUE TITLE
Problem with admin page display

### ISSUE THREAD

**Issue by HeartlandTechie** [thedevdojo/voyager#2751]
- Laravel Version: 5.6
- Voyager Version: 1.0
- PHP Version: PHP 7.1.8-1ubuntu1
- Database Driver & Version: MySQL 5.7.21-0ubuntu0.17.10.1

### Description:
Page column widths on fluid container not displaying correctly?

### Steps To Reproduce:

Use the system on Safari / Chrome / Firefox, all do the same results:

![screen shot 2018-03-01 at 9 48 48 am](https://user-images.githubusercontent.com/1760503/36854249-2581c614-1d36-11e8-8c17-6127ba41fb1e.png)
![screen shot 2018-03-01 at 9 49 13 am](https://user-images.githubusercontent.com/1760503/36854250-259cda76-1d36-11e8-8530-556d190d5f43.png)

**Comment by fletch3555** [comment]
Voyager is not optimized for mobile-size displays.  We'll look into that for the future, but it's VERY low on our priority list.

**Comment by HeartlandTechie** [comment]
I'm not using it on mobile, this is on desktop views.

**Comment by HeartlandTechie** [comment]
![screen shot 2018-03-01 at 9 54 48 am](https://user-images.githubusercontent.com/1760503/36854448-a1fab30e-1d36-11e8-8b07-8bd4757130ec.png)

**Comment by fletch3555** [comment]
Your previous screenshots made it seem like a mobile-sized display.  Do you have any custom CSS or errors in your browser console? That's certainly not the normal behavior.

Also, please confirm your version numbers.  Voyager 1.0.* does not support Laravel 5.6.  Also, please provide the 3rd part for the voyager version.

**Comment by HeartlandTechie** [comment]
Sorry, Laravel 5.5.  I show no errors or issues on my end . . . I've played a bit expanding things out manually by CSS manipulation.  It's very odd I think.

**Comment by HeartlandTechie** [comment]
Just did a complete reinstall of Laravel 5.5 and did composer require tcg/voyager.  On initial creation of a BREAD page you will get the full width page, but then when you drop back to edit a BREAD page it will only give you the narrow unreadable version.  Likewise when you go to add/edit records the page is shrunken.  I'm using latest Safari on a Mac, but problem exists across Firefox/Chrome as well.

**Comment by HeartlandTechie** [comment]
I'm going to leave this one out here, because it's a weird fluke.  I couldn't figure out why when I added BREAD it looked fine, but then when I went back and edited it the page was squashed.  I pulled down a create new and and edit existing BREAD file and looked at them side by side to see if there was a difference in the files.   The issue is the edit is making a body class that is also the same name as your table file, and in my case (since I already hit the BradTest camel case thing of Laravel) I made my table "Widgets".   Soooo . . . . the widgets class already for the front page items was conflicting with my table name called widgets.  It's a really wild coincidence but perhaps changing how table names are put in the body class or making the actual widgets CSS class a different name would help this odd collision.  
![screen shot 2018-03-02 at 8 52 45 pm](https://user-images.githubusercontent.com/1760503/36929841-23ed4040-1e5c-11e8-881d-c63b503f39e9.png)
